### PR TITLE
Cypress. Adapting tests to the new filter functionality.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Adapting tests to the new filter functionality.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/support/commands_filters_feature.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+});
+
+Cypress.Commands.add('addFiltersRule', (groupIndex) => {
+    cy.сheckFiltersModalOpened();
+    cy.collectGroupID().then((groupIdIndex) => {
+        cy.get(`[data-id="${groupIdIndex[groupIndex]}"]`).contains('button', 'Add rule').click();
+    });
+});
+
+Cypress.Commands.add('setGroupConition', (groupIndex, contidion) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands_filters_feature.js:55]: setGroupCondition*

File: tests/cypress/support/commands_filters_feature.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        cy.get(`[data-id="${groupIdIndex[groupIndex]}"]`).within(() => {
+            cy.get('.group--header').first().trigger('mouseover');
+            cy.contains('button', contidion).click({ force: true });
+        });
+    });
+});
+
+Cypress.Commands.add(
+    'setFilter',
+    (groupIndex, ruleIndex, field, operator, valueSource, value, label, labelAttr, submit) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands_filters_feature.js:67]: Too many arguments. Can we use object with destructurization instead? 
```({groupIndex, ruleIndex, field, operator, valueSource, value, label, labelAttr, submit})```

cy.setFilter({groupIndex, ruleIndex, field, operator, valueSource, value, label, labelAttr, submit})

File: tests/cypress/support/commands_filters_feature.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands_filters_feature.js]: Thanks. Fixed.

File: tests/cypress/support/commands_filters_feature.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands_filters_feature.js]: Done. Reworked the test also.

**Comment by bsekachev** [comment]
Hi, @dvkruchinin 

Some tests failed on CI

**Pull request merged**